addon/doxywizard/doxywizard.cpp

```diff
@@ -263,6 +264,12 @@ void MainWindow::manual()
 
 void MainWindow::about()
 {
+  auto now = std::chrono::system_clock::now();
+  std::time_t time = std::chrono::system_clock::to_time_t(now);
+  auto current = *localtime(&time);
+  char tmp[10];
+  sprintf(tmp,"%d",current.tm_year+1900);
```

Comment:
Since we have a dependency on Qt here, I think it better to use `int year = QDate::currentDate().year();`

Reply:
I wanted to have it as close as possible to the version used in the doxygen main program so I used the construct as proposed, but I see the point. Adjusted.